articles/users/search/v3/migrate-search-v2-v3.md

```diff
@@ -65,7 +65,11 @@ Extension | Version with support for v3 | Considerations
 
 You can leverage the [logs](/logs) in the [Dashboard](${manage_url}/#/logs) to find calls to the `/api/v2/users` endpoint that use the User Search v2 engine, including calls performed by SDKs. Those logs will help you identify where code changes might be needed in your applications.
 
-Use the following query to retrieve all the logs related to User Search v2: `type:w AND description:*User*Search*v2*`. The logs will provide additional information in the description field, in the following cases:
+Use the following query to retrieve all the logs related to User Search v2: 
+- For Tenants in the **AU** or **EU** regions: `type:w AND description:*User*Search*v2*`
```

Comment:
This may be outdated soon as Tenant Logs migrate to other environments.
Please contact #sm-opex for more info and help to create a unique query filter for all environments.